packages/web-service/src/pages/common/yes-no.njk

```diff
@@ -0,0 +1,36 @@
+{% extends "standard-form.njk" %}
+{% from "radios/macro.njk" import govukRadios %}
+{% from "input/macro.njk" import govukInput %}
+
+{% set errorMap = {
+        'yes-no': {
+            'any.required': { ref: '#yes-no', text: noneSelected }
+        }
+    }
+%}
+
+{% block pageContent %}
+    {{ govukRadios({
+      idPrefix: "yes-no",
+      name: "yes-no",
```

Comment:
I inherently feel like having every yes/no form using the same named value for its POST data is going to cause problems somewhere down the line but can't put my finger on exactly why...

(So nothing to do here for now! 😄 )